src/lib/Libwin/passwd.c

```diff
@@ -2688,21 +2687,24 @@ getHomedir(char *user)
 	char	*homedir = NULL;
 	HANDLE	userlogin = INVALID_HANDLE_VALUE;
 	int	i;
+	struct passwd *pwdp;
```

Comment:
I feel, Its good to initialize NULL for avoiding dangling pointer related crashes due to a programming error.

Reply:
Done